frontend/src/views/camp/navigation/mobile/NavSidebar.vue

```diff
@@ -39,7 +39,7 @@
 
       <v-list v-if="!camp._meta.loading">
         <SidebarListItem
-          :title="camp.name"
+          :title="camp.title || camp.shortname"
```

Comment:
In the issue we said that the shortname should always be displayed. If we have place, we also show the title